notebooks/hello-universe/Regressing_TESS_rotation_periods_with_CNNs/Regressing_TESS_rotation_periods_with_CNNs.ipynb

```diff
@@ -0,0 +1,1019 @@
+      "Epoch:  18 [ 5500/ 8000 ( 69%)] Training Loss:  0.262553\n",
+      "Epoch:  18 [ 6000/ 8000 ( 75%)] Training Loss:  0.114818\n",
+      "Epoch:  18 [ 6500/ 8000 ( 81%)] Training Loss:  0.300890\n",
+      "Epoch:  18 [ 7000/ 8000 ( 88%)] Training Loss:  0.287287\n",
+      "Epoch:  18 [ 7500/ 8000 ( 94%)] Training Loss:  0.365015\n",
+      "Test loss: 0.2315\n",
+      "Epoch:  19 [    0/ 8000 (  0%)] Training Loss:  0.287767\n",
+      "Epoch:  19 [  500/ 8000 (  6%)] Training Loss:  0.460290\n",
+      "Epoch:  19 [ 1000/ 8000 ( 12%)] Training Loss:  0.209023\n",
+      "Epoch:  19 [ 1500/ 8000 ( 19%)] Training Loss:  0.134723\n",
+      "Epoch:  19 [ 2000/ 8000 ( 25%)] Training Loss:  0.304515\n",
+      "Epoch:  19 [ 2500/ 8000 ( 31%)] Training Loss:  0.134069\n",
+      "Epoch:  19 [ 3000/ 8000 ( 38%)] Training Loss:  0.255285\n",
+      "Epoch:  19 [ 3500/ 8000 ( 44%)] Training Loss:  0.233586\n",
+      "Epoch:  19 [ 4000/ 8000 ( 50%)] Training Loss:  0.105364\n",
+      "Epoch:  19 [ 4500/ 8000 ( 56%)] Training Loss:  0.357909\n",
+      "Epoch:  19 [ 5000/ 8000 ( 62%)] Training Loss:  0.313933\n",
+      "Epoch:  19 [ 5500/ 8000 ( 69%)] Training Loss:  0.256064\n",
+      "Epoch:  19 [ 6000/ 8000 ( 75%)] Training Loss:  0.131733\n",
+      "Epoch:  19 [ 6500/ 8000 ( 81%)] Training Loss:  0.286770\n",
+      "Epoch:  19 [ 7000/ 8000 ( 88%)] Training Loss:  0.227307\n",
+      "Epoch:  19 [ 7500/ 8000 ( 94%)] Training Loss:  0.347994\n",
+      "Test loss: 0.2306\n",
+      "Epoch:  20 [    0/ 8000 (  0%)] Training Loss:  0.220646\n",
+      "Epoch:  20 [  500/ 8000 (  6%)] Training Loss:  0.432503\n",
+      "Epoch:  20 [ 1000/ 8000 ( 12%)] Training Loss:  0.174004\n",
+      "Epoch:  20 [ 1500/ 8000 ( 19%)] Training Loss:  0.146219\n",
+      "Epoch:  20 [ 2000/ 8000 ( 25%)] Training Loss:  0.298285\n",
+      "Epoch:  20 [ 2500/ 8000 ( 31%)] Training Loss:  0.161496\n",
+      "Epoch:  20 [ 3000/ 8000 ( 38%)] Training Loss:  0.253883\n",
+      "Epoch:  20 [ 3500/ 8000 ( 44%)] Training Loss:  0.227627\n",
+      "Epoch:  20 [ 4000/ 8000 ( 50%)] Training Loss:  0.077674\n",
+      "Epoch:  20 [ 4500/ 8000 ( 56%)] Training Loss:  0.329031\n",
+      "Epoch:  20 [ 5000/ 8000 ( 62%)] Training Loss:  0.334313\n",
+      "Epoch:  20 [ 5500/ 8000 ( 69%)] Training Loss:  0.276307\n",
+      "Epoch:  20 [ 6000/ 8000 ( 75%)] Training Loss:  0.131502\n",
+      "Epoch:  20 [ 6500/ 8000 ( 81%)] Training Loss:  0.260511\n",
+      "Epoch:  20 [ 7000/ 8000 ( 88%)] Training Loss:  0.259440\n",
+      "Epoch:  20 [ 7500/ 8000 ( 94%)] Training Loss:  0.357995\n",
+      "Test loss: 0.2293\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Begin training:\")\n",
+    "\n",
+    "# Initialize CNN\n",
+    "model = ConvNet()\n",
+    "\n",
+    "# Train CNN\n",
+    "weights, train_p_loss, val_p_loss = train(model, train_loader, val_loader,\n",
+    "    early_stopping_patience=10, num_epochs=num_epochs)"
```

Comment:
Maybe here or in a follow-on cell you should plot the loss over epochs?  It will help the reader understand how things are evolving (especially given you say things below about it)